airbyte-integrations/bases/base-normalization/integration_tests/config.json

```diff
@@ -0,0 +1 @@
+{"destination_path": "/local/test.duckdb"}
```

Comment:
formatter seems to want a newline at the end of this file (probably easier to just `SUB_BUILD=CONNECTORS_BASE ./gradlew format`)